KoreBuild-dotnet/build/KoreBuild.cmd

```diff
@@ -0,0 +1,31 @@
+@ECHO off
```

Comment:
For the second one, wouldn't it be changing the build script inside Sake? I'd much rather have us use Sake \ C# for tweaking rather than have to manage cmd and sh files.